include/ttmlir/Target/Utils/MLIRToFlatbuffer.h

```diff
@@ -29,6 +31,28 @@ inline ::tt::target::OOBVal toFlatbuffer(FlatbufferObjectCache &,
   }
 }
 
+inline std::uint64_t getElementSizeBytes(DataType dtype) {
```

Comment:
I hear your point, and I might be swayed one way or the other, but currently isn't any shared code between compiler and runtime except for FBS.  It is somewhat deliberate in that it might be good to leave the door open for one day where the flatbuffer runtime grows out of scope of tt-mlir and becomes its own repo.